src/App/Resources/AppResources.resx

```diff
@@ -1972,4 +1972,10 @@
     <value>About Send</value>
     <comment>'Send' is a noun and the name of a feature called 'Bitwarden Send'. It should not be translated.</comment>
   </data>
+  <data name="HideEmail" xml:space="preserve">
+    <value>Hide my email address from recipients.</value>
+  </data>
+  <data name="SendOptionsPolicyInEffect" xml:space="preserve">
+    <value>One or more organization policies are affecting your Send options.</value>
```

Comment:
Strings containing 'Send' should also include the `<comment>` about Send not being translated (see above)